tools/cli/src/commands/task_queue.rs

```diff
@@ -89,9 +89,10 @@ impl TaskQueue {
             )
             .await?;
 
-        let event: TaskCreatedEvent = CosmosTxEvents::from(&tx_resp)
+        let event: cosmwasm_std::Event = CosmosTxEvents::from(&tx_resp)
             .event_first_by_type(TaskCreatedEvent::NAME)?
-            .try_into()?;
+            .into();
+        let event: TaskCreatedEvent = event.try_into()?;
```

Comment:
This looks better. The cli can obviously care about Climb and all. And this translation makes sense